apps/open-swe/langbench/evaluator.eval.ts

```diff
@@ -28,34 +30,45 @@ logger.info(`Starting evals over ${DATASET.length} PRs...`);
  */
 async function processPR(prData: PRData): Promise<PRProcessResult> {
   const result: PRProcessResult = {
-    pr_number: prData.pr_number,
-    repo_name: prData.repo_name,
+    prNumber: prData.prNumber,
+    repoName: prData.repoName,
     success: false,
-    evals_found: false,
-    evals_files: [],
+    evalsFound: false,
+    evalsFiles: [],
+    testFiles: [],
   };
   const daytona = new Daytona({
     organizationId: process.env.DAYTONA_ORGANIZATION_ID,
   });
   let sandbox: Sandbox | undefined;
 
   try {
-    logger.info(`Processing PR #${prData.pr_number}: ${prData.title}`);
+    logger.info(`Processing PR #${prData.prNumber}: ${prData.title}`);
 
+    // Use test files from PR data (already fetched and stored)
+    const testFiles = prData.testFiles || [];
+    result.testFiles = testFiles;
     // Create sandbox
     sandbox = await daytona.create(DEFAULT_SANDBOX_CREATE_PARAMS);
-    result.workspace_id = sandbox.id;
 
+    // Validate sandbox was created properly
+    if (!sandbox || !sandbox.id) {
+      throw new Error(
+        `Failed to create valid sandbox. Sandbox object: ${JSON.stringify(sandbox)}`,
```

Comment:
i wouldn't log the sandbox. if it's undefined, just throw err
```suggestion
        "Failed to create valid sandbox",
```